lucien-ios/AddAndEditNewComicBook/ComicFormViewModel.swift

```diff
@@ -122,7 +126,13 @@ class ComicFormViewModel {
                 else { return }
 
             comic.asObservable().subscribe(onNext: { comic in
-                self?.addComicBook(comic: comic, completion: completion)
+                if self?.comicFormMode == .add {
+                    self?.addComicBook(comic: comic, completion: completion)
+                } else {
+                    guard let id = self?.comicID else { return }
+                    self?.editComicBook(id: id, comic: comic, completion: completion)
+                }
+                
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Trailing Whitespace Violation: Lines should not have trailing whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Atrailing_whitespace)